expect.yaml

```diff
@@ -0,0 +1,64 @@
+# Partly generated from https://git.alpinelinux.org/aports/plain/main/expect/APKBUILD
+package:
+  name: expect
+  version: 5.45.4
+  epoch: 0
```

Comment:
This is over 6 years old. I'm not sure how this can be security supported, as it feels like EOL by definition.

Meaning, any security patches would have to be on us and/or coordinated with other distributions.